R/utils_tfrmt.R

```diff
@@ -23,26 +28,49 @@ apply_tfrmt <- function(.data, tfrmt, mock = FALSE){
     values = tfrmt$values,
     column = tfrmt$column,
     mock = mock
-  )
+  ) %>%
+    tentative_process(apply_col_align_plan, tfrmt$col_align,
+                      tfrmt$column, tfrmt$values,
+                      fail_desc= "Unable to align dataset")
+
+  ## append span structures to dataset for handling post-this function
+  if(!is.null(tfrmt$col_plan$span_structures)){
```

Comment:
any reason why we don't use tentative_process here?

Reply:
Because it takes in a whole tfrmt so would try to work on any tfrmt rather than just if there is a span_structure